traffic_ops/traffic_ops_golang/login/login.go

```diff
@@ -21,8 +21,13 @@ package login
 
 import (
 	"encoding/json"
+	"errors"
 	"fmt"
+	"github.com/dgrijalva/jwt-go"
```

Comment:
Did you get a chance to look into whether or not we could use one of these dependencies over the other, or do you think we still need them both?

For whatever we choose, we should probably run them by the mailing list just to make sure nobody has any issues with including those dependencies in the project. Once they're agreed upon, we typically vendor our go dependencies within our repo (probably in the traffic_ops_golang/vendor dir).

Reply:
Also, do these have Apache-compatible licenses?   This shouldn't be merged until we know that for sure.

Reply:
I believe they are Apache-compatible. They're both licensed under the MIT License, which - if I'm reading [the 3<sup>rd</sup>-party guidelines correctly](https://apache.org/legal/resolved.html#category-a) - are "similar" to Apache 2.0

Reply:
I tried to cut it down to just one of them but its not trivial / maybe not possible. i can respond to my email in the distro and point out that i added them in.

Reply:
Ok, once the mailing list gives the thumbs up on these new dependencies, we will have to vendor them in our project. I would recommend doing that in a separate PR that just vendors these dependencies. Once that's merged, this PR can merge. Basically, we just `git clone` their repo into one of our vendor directories, and remove their `.git` directory so that git doesn't think it's a sub-repo.